src/AccessibilityInsights.Extensions.AzureDevOps/ConnectionInfo.cs

```diff
@@ -45,16 +45,8 @@ public class ConnectionInfo
         public ConnectionInfo(Uri serverUrl, TeamProject project, AdoTeam team)
         {
             ServerUri = serverUrl;
-            if (project != null)
-            {
-                TeamProject typedProject = project as TeamProject;
-                Project = typedProject ?? new TeamProject(project);
-            }
-            if (team != null)
-            {
-                AdoTeam typedTeam = team as AdoTeam;
```

Comment:
Since team is already an AdoTeam, typedTeam will be identical to team, which is non-null. As such, we can simply the assignment of Team.